src/app/components/category-card/category-card.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="card" style="margin: auto">
+    <form [formGroup]="categoryForm">
+      <div class="loading-indicator" *ngIf="loading === true">
+        <mat-progress-spinner
+          mode="indeterminate"
+          color="accent"
+        ></mat-progress-spinner>
+      </div>
+      <div class="cat-Action">
+        <button
+          class="buttonCat"
+          (click)="onEditClick()"
+          *ngIf="editMode === false"
+        >
+          <mat-icon>edit</mat-icon>
+        </button>
+        <button
+          (click)="onDelete()"
+          *ngIf="editMode === false"
+          [ngClass]="{
+            disabledButton: canMakeChanges === false,
+            buttonCat: canMakeChanges === true
+          }"
+          [disabled]="canMakeChanges ? false : true"
+        >
+          <mat-icon>delete</mat-icon>
+        </button>
+        <button
+          class="buttonCat"
+          *ngIf="editMode === true"
+          (click)="onSubmitForm()"
+        >
+          <mat-icon>save</mat-icon>
+        </button>
+        <button class="buttonCat" *ngIf="editMode === true" (click)="onDiscard()">
+          <mat-icon>close</mat-icon>
+        </button>
+      </div>
+  
+      <div class="segment flex-center">
+        <mat-form-field appearance="fill" style="width: 50%">
+          <mat-label> Title. </mat-label>
+          <input
+            matInput
+            [readonly]="canMakeChanges && editMode ? null : true"
+            required
```

Comment:
This attribute is used with template driven forms, not reactive ones. You can read more about template driven forms and reactive forms here: https://blog.angular-university.io/introduction-to-angular-2-forms-template-driven-vs-model-driven/#:~:text=Template%20Driven%20Forms%20are%20based,powerful%20and%20easier%20to%20use.